client/src/components/pages/addFlat/AddFlat.jsx

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react';
+import Input from '../../../common/input/Input.jsx';
+import Button from '../../../common/button/Button.jsx';
+import ImageUploader from 'react-images-upload';
+import './addFlat.scss';
+
+class AddFlat extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
```suggestion
    this.initialState = { ...the same that u have for state for now }
    this.state = {
        ...initialState
    }
```

and then u can use initial state for refreshing state below in onSubmit func